spec/fortnox/api/circular_queue_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'fortnox/api/circular_queue'
+
+describe Fortnox::API::CircularQueue do
+  describe 'start index' do
+    context 'when running several times' do
+      let(:samples) { Array.new(100) { described_class.new(*(0..99)).next } }
```

Comment:
`(0..99).to_a` is more explicit and right now you are creating the same array of a hundred integers 100 times, pull this into its own `let` and reuse the same array for all samples.